### ISSUE TITLE
FR: Reduce the size of all the turmoil components

### ISSUE THREAD

**Issue by kberg** [bafolts/terraforming-mars#2779]
The Turmoil components are larger than the map! (That's from memory, I could be wrong.) I bet the whole thing could drop about 35-50% in both dimensions and be just as playable.

**Comment by vsrisuknimit** [comment]
@kberg I agree with Turmoil components in general can be smaller. The problem however is that Turmoil has text on them. It can be hard to read the global event text if we simply scale down the whole thing by 50%. May need an alternative solution.

**Comment by ssimeonoff** [comment]
keep in mind that turmoil board is at 80% at the moment

**Comment by Borbarad2** [comment]
Is this still an issue? or could the issue be closed?
